From: Jakub Jirutka <jakub@jirutka.cz>
Date: Wed, 23 Mar 2016 21:28:00 +0000
Subject: [PATCH] fix dependencies

uWSGI is installed from Alpine package, but pip doesn't know about it,
so it tries to install it from PyPI, but compilation fails.
--- a/setup.py
+++ b/setup.py
@@ -129,7 +129,6 @@
     'toronado>=0.0.4,<0.1.0',
     'ua-parser>=0.6.1,<0.7.0',
     'urllib3>=1.14,<1.15',
-    'uwsgi>2.0.0,<2.1.0',
     'rb>=1.4.0,<2.0.0',
 ]
 
